mex/extractors/publisher/filter.py

```diff
@@ -0,0 +1,25 @@
+from collections.abc import Generator, Iterable
+
+from mex.common.models import MergedItem
+from mex.extractors.logging import log_processed_merged_items
+from mex.extractors.settings import Settings
+
+
+def filter_merged_items(
+    items: Iterable[MergedItem],
+) -> Generator[MergedItem, None, None]:
+    """Filter to be published items by allow list."""
+    settings = Settings.get()
+
+    logging_counter = 0
+    total_counter = 0
```

Comment:
```suggestion
    skipped_items = 0
    total_items = 0
```
maybe that'd be a bit clearer?